### PR TITLE
[Fix] Fix TotalText Anno version issue

### PR DESCRIPTION
**Pull request by xinke-wang**
## Motivation

1. To solve issue #940 
2. Fix some typos in docs

### CODE REVIEW

File: tools/data/textrecog/totaltext_converter.py [hunk 1]
```diff
@@ -15,13 +15,14 @@
...
+def collect_files(img_dir, gt_dir):
     """Collect all images and their corresponding groundtruth files.
 
     Args:
-        img_dir(str): The image directory
-        gt_dir(str): The groundtruth directory
-        split(str): The split of dataset. Namely: training or test
+        img_dir (str): The image directory
+        gt_dir (str): The groundtruth directory
+        split (str): The split of dataset. Namely: training or test
```
Comment (Mountchicken) [inline tools/data/textrecog/totaltext_converter.py:24]:
```suggestion
```

Comment (Mountchicken) [inline tools/data/textrecog/totaltext_converter.py:24]:
```suggestion
```

File: tools/data/textrecog/totaltext_converter.py [hunk 2]
```diff
@@ -345,30 +352,21 @@ def load_img_info(files):
...
-    out_dir = args.out_dir if args.out_dir else root_path
-    mmcv.mkdir_or_exist(out_dir)
-
     img_dir = osp.join(root_path, 'imgs')
     gt_dir = osp.join(root_path, 'annotations')
 
     set_name = {}
-    for split in args.split_list:
+    for split in ['training', 'test']:
         set_name.update({split: 'instances_' + split + '.json'})
```
Comment (Mountchicken) [inline tools/data/textrecog/totaltext_converter.py:370]: The annotation files generated in the code are `train_label.txt` and `test_label.txt`, but the print values here are `instance_training.json` and `nstance_test.json`
should we use 
```python
set_name.update({split: split+'_label'+'.txt'})
```

Comment (Mountchicken) [inline tools/data/textrecog/totaltext_converter.py:370]: The annotation files generated in the code are `train_label.txt` and `test_label.txt`, but the print values here are `instance_training.json` and `nstance_test.json`
should we use 
```python
set_name.update({split: split+'_label'+'.txt'})
```

File: tools/data/textrecog/totaltext_converter.py (hunk 1 above)
Comment (xinke-wang) [reply tools/data/textrecog/totaltext_converter.py]: deleted

File: tools/data/textrecog/totaltext_converter.py (hunk 2 above)
Comment (xinke-wang) [reply tools/data/textrecog/totaltext_converter.py]: done

**Review by Mountchicken — APPROVED** [review]

File: tools/data/textrecog/totaltext_converter.py [hunk 3]
```diff
@@ -152,15 +156,16 @@ def process_line(line, contours, words):
...
+        words (list[list]): A list of lists of words (string)
+            for the text instances
+
     Returns:
-        contours(list[lists]): A list of lists of contours
+        contours (list[lists]): A list of lists of contours
         for the text instances
-        words(list[list]): A list of lists of words (string)
+        words (list[list]): A list of lists of words (string)
         for the text instances
```
Comment (gaotongxiao) [inline tools/data/textrecog/totaltext_converter.py:169]: indentation

**Review by gaotongxiao — COMMENTED** [review]
Did we test these scripts on the latest totaltext?

**Pull request merged**